apps/src/templates/sectionsRefresh/SingleSectionSetUp.jsx

```diff
@@ -2,15 +2,20 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import MultiSelectGroup from '@cdo/apps/templates/teacherDashboard/MultiSelectGroup';
 import {StudentGradeLevels} from '@cdo/apps/util/sharedConstants';
+import {queryParams} from '@cdo/apps/code-studio/utils';
 import moduleStyles from './sections-refresh.module.scss';
 import i18n from '@cdo/locale';
 
 export default function SingleSectionSetUp({
   sectionNum,
   section,
   updateSection,
+  isNewSection,
 }) {
   const gradeOptions = StudentGradeLevels.map(g => ({label: g, value: g}));
+  const participantType = isNewSection
+    ? queryParams('participantType')
+    : section.participantType;
```

Comment:
I mentioned this in the comments, but do we know for a fact if every section has a "participantType"?  Or do we want to account for that here?

Reply:
We talked about this offline but I'll note it here as well! Every existing section has a participant type (it's a non-null field in the model) and new sections are required to select one that will show up as a query param!